src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -1209,68 +1212,77 @@ private async Task OnValueChanged(long index, string type)
                     LicenseManager.ReloadLicenseLimits();
                     ConcurrentBackupsCounter.ModifyMaxConcurrentBackups();
                     NotifyAboutClusterTopologyAndConnectivityChanges();
-
-                    break;
-
-                case nameof(PutServerWideBackupConfigurationCommand):
-                    RescheduleTimerForIdleDatabases(index);
                     break;
             }
         }
 
-        private void RescheduleTimerForIdleDatabases(long taskId)
+        private void RescheduleTimerIfDatabaseIdle(string db, object state)
```

Comment:
Is it called per database or only once?
Maybe we can send the index for that database and not all the indexes for all databases?

Reply:
Then if the state is null we don't need to do anything.

Reply:
Done